### PR TITLE
I feel... Like Crying.

### PR DESCRIPTION
**Pull request by kkrosie123**
Please don’t murder me with how short this is - I’m not done yet, and it’s 2:30 in the morning and I need sleep :”D

But while I still had my computer I had programmed this and a bunch of other moods into the game. This one in particular was very special to me, and I’d thought I’d put it out here for other people as well.

If this doesn’t fit, that’s okay. Maybe I can make it a submod or something. It serves a similar purpose to sad, but it just felt more therapeutic to me somehow.

### CODE REVIEW

File: Monika After Story/game/script-moods.rpy [hunk 1]
```diff
@@ -668,3 +668,38 @@ label mas_mood_bored:
...
+
+init 5 python:
+    addEvent(Event(persistent._mas_mood_database,eventlabel="mas_mood_crying",prompt="...like crying.",category=[store.mas_moods.TYPE_BAD],unlocked=True),code="MOO")
+    
+label mas_mood_crying:
+    m "[player]!"
+    m "Are you okay?{nw}"
+    $ _history_list.pop()
+    menu:
+        m "Are you okay?{fast}"
```
Comment (multimokia) [inline Monika After Story/game/script-moods.rpy:680]:
```suggestion
        m "Are you okay?{fast}"

```

File: Monika After Story/game/script-moods.rpy [hunk 2]
```diff
@@ -668,3 +668,38 @@ label mas_mood_bored:
...
+        "Yes.":
+            m "Okay, good. That's a relief."
+            m "I'm here to keep you company and you can talk to me if you need anything, okay?"
+            m "I love you [player]. You're my everything."
+            return "love"
```
Comment (multimokia) [inline Monika After Story/game/script-moods.rpy:685]:
```suggestion
            $ mas_ILY()

```

File: Monika After Story/game/script-moods.rpy [hunk 3]
```diff
@@ -668,3 +668,38 @@ label mas_mood_bored:
...
+        "No.":
+            m "..."
+            m "[player]..."
+            m "I'm so sorry. Did something happen...?"
+            m "I love you so much [player], and if you need to talk about something, I'm right here."
+            m "So... do you need
```
Comment (multimokia) [inline Monika After Story/game/script-moods.rpy:691]:
```suggestion
            m "So... do you need"

```

File: Monika After Story/game/script-moods.rpy [hunk 4]
```diff
@@ -668,3 +668,38 @@ label mas_mood_bored:
...
+        "I'm not sure.":
+            m "[player]..."
+            m "Did something happen? Is everything okay?"
+            m "If you need to talk about something, I'm right here.{nw}"
+            $ _history_list.pop()
+            menu:
+                m "If you need to talk about something, I'm right here.{fast}"
+                "Everything is fine.":
```
Comment (multimokia) [inline Monika After Story/game/script-moods.rpy:699]:
```suggestion

                "Everything is fine.":
```

File: Monika After Story/game/script-moods.rpy [hunk 5]
```diff
@@ -668,3 +668,38 @@ label mas_mood_bored:
...
+                    m "... Okay, [player]. I'll be here if you change your mind. I love you~"
+                    return "love"
+                "Can I talk to you?":
```
Comment (multimokia) [inline Monika After Story/game/script-moods.rpy:702]:
```suggestion

                "Can I talk to you?":
```

File: Monika After Story/game/script-moods.rpy [hunk 6]
```diff
@@ -668,3 +668,38 @@ label mas_mood_bored:
...
+                    m "Yes, of course [player]."
+                
+return
```
Comment (multimokia) [inline Monika After Story/game/script-moods.rpy:705]:
```suggestion
    return
```

File: Monika After Story/game/script-moods.rpy [hunk 7]
```diff
@@ -668,3 +668,57 @@ label mas_mood_bored:
...
+        "I'm not sure.":
+            m "[player]..."
+            m "Did something happen? Is everything okay?"
+            call mas_mood_uok
+      m "Well..."
+      m "If you do end up crying..."
+      m "I hope it helps you, [player]."
+      m "There's nothing wrong with crying, okay?"
+      m "You're wonderful, [player]. You can cry as much as you need to."
+      m "I love you, and I always will."
```
Comment (multimokia) [inline Monika After Story/game/script-moods.rpy:700]:
```suggestion

    m "Well..."
    m "If you do end up crying..."
    m "I hope it helps you, [player]."
    m "There's nothing wrong with crying, okay?"
    m "You're wonderful, [player]. You can cry as much as you need to."
    m "I love you, and I always will."
```

File: Monika After Story/game/script-moods.rpy [hunk 8]
```diff
@@ -668,3 +668,58 @@ label mas_mood_bored:
...
+    $ _history_list.pop()
+    menu:
+        m "If you need to talk about something, I'm right here.{fast}"
+        "I'd like to talk with you.":
+            m "Go ahead, [player]."
+            m "...{nw}"
+            $ _history_list.pop()
+            menu:
+                m "...{fast}"
+                "I'm done."
```
Comment (multimokia) [inline Monika After Story/game/script-moods.rpy:716]:
```suggestion
                "I'm done.":
```

**Comment by kkrosie123** [comment]
Thank you multi for making my topic functional ily

**Pull request merged**